### ISSUE TITLE
Use SCSS

### ISSUE THREAD

**Issue by iangilman** [ContextInstitute/bfcom#24]
It would be great if we could write SCSS directly and have the site build it for us.

This looks like a potential solution:

https://wordpress.org/plugins/wp-scss/

**Comment by rgilman** [comment]
Would you be willing to install that on your local instance as a test? My lean at the moment is to keep the master branch and bfn.context.org using css but allow each developer to work in SCSS or not.

**Comment by iangilman** [comment]
Sure, I can give it a try first, but no, we all have to adopt SCSS or none of us can. SCSS gets compiled to CSS, but if anyone changes the resulting CSS, those changes will only last until the next time the SCSS gets compiled.

**Comment by rgilman** [comment]
Let's go with adopting SCSS. Do you have any thoughts about segmenting our SCSS into various files or shall we just start with one?

**Comment by rgilman** [comment]
I would, however, like the workflow to include compiling the SCSS before pushing to GitHub (master or branch). How does that sound to you?

**Comment by iangilman** [comment]
It's generally not a good idea to include built files in GitHub at all. It's too easy for them to get out of sync, and even if you are good about always updating them, it creates a lot of unnecessary noise in every commit. The cleanest way to do it is to have the build be a part of everyone's local process, and also part of the deploy. 

As for breaking the SCSS into multiple files, yes, I'm sure we'll find a nice way to keep each file small, with styles for just one component each or something.

**Comment by rgilman** [comment]
I'm still not clear on the workflow. If I build locally do I then exclude the CSS files with a .gitignore?

**Comment by iangilman** [comment]
Yes, that's the idea. We can't do that until we have a proper build process set up for everyone, of course.

**Comment by rgilman** [comment]
So for now, until we get that proper build process set up for everyone, we should include the CSS in the repo?
